plugins/image.yaml

```diff
@@ -0,0 +1,57 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: image
+spec:
+  version: v0.1.0
+  homepage: https://github.com/chenjiandongx/kubectl-image
+  shortDescription: Show container images used in the cluster.
+  description: |
+    This plugin shows container images used in the Kubernetes cluster in a
+    table view. You can show all images or show images used in a specified
+    namespace.
+
+    To display a table of all images in current namespace:
+      $ kubectl image
+
+    To display a table of images in dev namespace:
+      $ kubectl image -n dev
+
+    To display a table of images in all namespaces:
+      $ kubectl image --all-namespaces
```

Comment:
This part is not necessary.

Reply:
Agreed, these examples do not add anything about _what_ this plugin does. This should be moved to the `--help` output.